src/styl/imports/card.styl

```diff
@@ -69,3 +69,83 @@
 			transform rotate(3deg) scale(1.05)
 		a
 			border none
+
+		label::before
+			border-color transparent
+
+	&[data-completed='true']
+		& > *:not(label)
+			opacity 0.4
+
+		label::before
+			content: none
+
+	label
+		float right
+		display flex
+		flex-direction row-reverse
+		position relative
+		cursor pointer
+
+		&::before
+			content 'Mark as complete?'
```

Comment:
This isn't accessible. Also, this will compete for space with the `Date Added` coming in #328. I think this should be centered in a new row. I have a feeling #407 will move this metadata to below the picture, but don't worry too much about that